File: back/src/main/java/csh/back/domain/vote/vote/dto/web/VoteTimelineResponse.java
package csh.back.domain.vote.vote.dto.web;

import csh.back.domain.trip.timeline.entity.Timeline;

public record VoteTimelineResponse(
        Long confirmPlaceId,
        Timeline timeline
) {
    public static VoteTimelineResponse of(Long confirmPlaceId, Timeline timeline) {
        return new VoteTimelineResponse(confirmPlaceId, timeline);
    }
}

File: back/src/main/kotlin/csh/back/domain/vote/vote/dto/web/VoteTimelineResponse.kt
package csh.back.domain.vote.vote.dto.web

import csh.back.domain.trip.timeline.entity.Timeline

data class VoteTimelineResponse(
    val confirmPlaceId: Long,
    val timeline: Timeline,
) {
    companion object {
        @JvmStatic
        fun of(confirmPlaceId: Long, timeline: Timeline): VoteTimelineResponse =
            VoteTimelineResponse(confirmPlaceId, timeline)
    }
}
